### ISSUE TITLE
Manilla

### ISSUE THREAD

**Issue by missinglink** [whosonfirst-data/whosonfirst-data#1378]
Heya,

The capital city of The Philippines "Manilla" (421168919) has been superseded by "Metro Manilla" (1159397249).

The new record includes the prefix "Metro" which is problematic for exact-matching search queries.

Could we please either:
- Add additional name/names of simply "Manilla" without the prefix.
- Update the existing names to remove the "Metro" part from the names.

Is there a WOF policy regarding these sort of things?
Eg. Should 'Kansas' be called 'Kansas State' and 'Berlin' as 'Stadt Berlin'?

It seems cleaner to omit the local name for the placetype and instead use `wof:placetype_local` to say this is a `metro`?

**Comment by nvkelso** [comment]
I can see changing the preferred name to just Manilla here – we generally go with the more conventional spelling, like in London and a few other places. But @stepps00 and I will need to look at the hierarchy here to figure out the exact details when he's back.

There's a separate discussion around `Kansas` versus `Kansas State` that involves a new name property / and I think that's different again from `placetype_local`.

**Comment by stepps00** [comment]
Yeah, I think the correct thing to do here is change the "Metro Manila" name to just "Manila".

Metro Manila refers to the National Capital District, which WOF stores as a region record, [here](https://spelunker.whosonfirst.org/id/1159397243/). The "Metro Manila" record, [here](https://spelunker.whosonfirst.org/id/1159397249/), is a locality record within that region.

WOF already has "Metro Manila" as a name variant in the record for the National Capital District, and sources confirm that stripping out "Metro" from the locality record name is the correct thing to do here.

- https://en.wikipedia.org/wiki/Metro_Manila
- https://en.wikipedia.org/wiki/Metro_Manila#/media/File:Metro_Manila_in_the_Philippines.png
- http://statoids.com/uph.html
